src/midisequencer.h

```diff
@@ -44,6 +44,7 @@ namespace midisequencer{
         uint_least32_t message;
         int port;
         int track;
+        float ticks;
```

Comment:
Does this need to be a float? Whenever I think of a concept of a number of ticks, it's always integral.

Using a float means we can accumulate rounding error and mysterious inaccuracies that are hard to track down.

Reply:
at least time (float) and time_offset (double) are, but not sure if actually required